gradle-plugins/compose/src/test/test-projects/application/proguard/rules.pro

```diff
@@ -1,3 +1,4 @@
 -keep public class Main {
     public void keptByKeepRule(...);
-}
\ No newline at end of file
+}
+-dontnote
```

Comment:
Shouldn't we ignore just some notes, related to our libraries and Kotlin?

Some notes can be helpful

Reply:
I realized that it is a test project :)